src/data/teams.json

```diff
@@ -641,6 +641,19 @@
           "profileImage": "https://avatars0.githubusercontent.com/u/32864116?s=460&u=0c2a53c6f1476da904d2b3c820bab43b64b7185b&v=4",
           "slackUsername": "James Pang"
         }
+      },
+      {
+        "title": "WordPress Developer",
+        "isLead": false,
+        "payPerDay": 2800,
+        "createdDate": "2020-07-08T02:14:59.307535Z",
+        "contributor": {
+          "contributorId": "U01H0QWK2BG",
+          "displayName": "Aathil Ahamed",
+          "githubUsername": "aathil-Mr-ITGuy",
+          "profileImage": "https://ca.slack-edge.com/T07PJD1FZ-U01H0QWK2BG-7b8970a84b6c-512",
```

Comment:
use this as his profile image url instead: `https://avatars0.githubusercontent.com/u/36620321?s=400&u=af0419d3610824677b5f529008f9f87b23623091&v=4`